test/spec/utils.js

```diff
@@ -0,0 +1,16 @@
+exports.getSearchCount = function*() {
```

Comment:
'function*' is only available in ES6 (use esnext option).<br>'exports' is not defined.